css/cssom/getComputedStyle-pseudo.html

```diff
@@ -105,24 +105,41 @@
   <ul><li id="pseudo-invalid">Item</li></ul>
 </div>
 <script>
-test(function() {
-  var div = document.getElementById('test');
-  [":before", ":after"].forEach(function(pseudo) {
+test(() => {
+  const div = document.getElementById('test');
+  ["before", "after"].forEach(pseudo => {
+    assert_equals(getComputedStyle(div, pseudo).width, "100px");
```

Comment:
```suggestion
    assert_not_equals(getComputedStyle(div, pseudo).width, "100px");
```

If this is invalid (per subtest name), shouldn't this be not_equals ? or equals to empty string?

Reply:
getComputedStyle has this weird thing that when its pseudo-element does not start with a colon it's supposed to be ignored.

Reply:
Ah yeah I see this in the algorithm, perhaps you could make the subtest message more clear then, or add a comment to clearly distinguish the invalid case and the no-colon case